staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -253,6 +270,13 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 		go apiApprovalController.Run(5, context.StopCh)
 		go finalizingController.Run(5, context.StopCh)
 
+		if utilfeature.DefaultFeatureGate.Enabled(features.StorageVersionAPI) &&
+			utilfeature.DefaultFeatureGate.Enabled(features.APIServerIdentity) {
+			// this goroutine will just monitor when its time for the storageversion
+			// manager to shutdown its queues.
+			go crdHandler.storageVersionManager.Shutdown(context.StopCh)
```

Comment:
It seems unfortunate to have to run a goroutine to block on that stop channel only to close another stop channel when it closes. Is there any clean way to pass this context.StopCh down and use it in the storageVersionManager?